packages/react-components/src/entities/EventSidebar/details/Measurements.js

```diff
@@ -9,18 +9,23 @@ export function Measurements({ data }) {
     const [fixedColumn, setFixed] = useState(true);
 
     let hasMeasurements = false;
-    if (data.results.documents.results
-        && data.results.documents.results.length > 0
-        && data.results.documents.results[0].measurementOrFacts
-        && data.results.documents.results[0].measurementOrFacts.length > 0) {
+    if (data.documents.results
+        && data.documents.results.length > 0
+        && data.documents.results[0].measurementOrFacts
+        && data.documents.results[0].measurementOrFacts.length > 0) {
         hasMeasurements = true;
     }
 
     if (!hasMeasurements){
         return <></>
```

Comment:
This is preference I suppose, but isn't returning `null` more clear?

Reply:
this has been done by @qifeng-bai